trussworks/react-uswds
# chore: migrate to storybook 8

# Summary
Makes the minor changes needed to update to Storybook 8.

- sb dependencies updated
- happo sb plugin updated (sb 8 support)
- mdx-gfm sb addon added
- remove `.stories` part of welcome mdx name (sb 8 ignores `.stories.mdx` files)
- use `blocks` in welcome mdx

## How To Test
- Run `storybook` script to launch storybook dev
- Verify storybook works without error
